src/rendering/vcSceneLayerRenderer.cpp

```diff
@@ -32,8 +32,7 @@ udResult vcSceneLayerRenderer_Create(vcSceneLayerRenderer **ppSceneLayerRenderer
 epilogue:
   if (pSceneLayerRenderer != nullptr)
   {
-    vcSceneLayer_Destroy(&pSceneLayerRenderer->pSceneLayer);
-    udFree(pSceneLayerRenderer);
```

Comment:
Does this happen somewhere else now? I feel like I added this because it leaks in certain situations?

Reply:
No I don't believe so. I just changed this because it's more consistent with our error handling. I will revert it however, not worth the risk.